### ISSUE TITLE
Whitespaces are condensed to one character in json-timeline

### ISSUE THREAD

**Issue by YamatoSecurity** [Yamato-Security/hayabusa#1048]
@hitenkoku Could you take a look at this?
Multiple whitespaces are being condensed to just a single whitespace with `json-timeline`.
<img width="354" alt="Screen Shot 2023-05-14 at 9 22 53 AM" src="https://github.com/Yamato-Security/hayabusa/assets/71482215/c548344e-800b-4188-b4bb-5beea969337d">

Original JSON file:
<img width="1269" alt="Screen Shot 2023-05-14 at 9 25 37 AM" src="https://github.com/Yamato-Security/hayabusa/assets/71482215/d6b210b2-a565-4b71-a972-c0c1fe1c5e13">

There should be 4 spaces between `\r\n` and `\"Expand-7Zip\"`.

I made the json timeline with this:
`./hayabusa-2.5.0-win-x64.exe json-timeline -f ./WELA-challenge-2023-05-13.json -D -u -U -n -o WELA-challenge.json -J`

and then extracted them out with this:
`cat WELA-challenge.json | jq 'select ( .EventID == 4104 and .Details.ScriptBlock? != "n/a" and  .Computer == "UCHIHA-PC.hinokabe.local" ) | .Details.ScriptBlock , "\r\n"' -j | awk '{ gsub(/\\r\\n/,"\r\n"); print; }' | awk '{ gsub(/\\t/, "\t"); print; }' | awk '{ gsub(/\\n/, "\r\n"); print; }' > 4104-PowerShell-Logs-Uchiha.ps1`

You can use the `kosen-challenge.json` file from the TMCIT WELA contest.
I can send it over to you on slack if you need it.

**Comment by hitenkoku** [comment]
@YamatoSecurity 

Confirmed, when we break down the contents of Details, we will do a blank split and the original blank data will be replaced by a single blank data

It takes a lot of work to reproduce the whitespace used in the original Details.

**Comment by hitenkoku** [comment]
I checked again and the data obtained from the following command (no jq) seems to show 4 blanks. Could you please confirm this?

- Command
```
> ./main.exe json-timeline -d ../hayabusa-sample-evtx -D -u -U -n -f ./kosen-challenge.json -J -o 1048.json
```

- Result (cat main.json)
```
{
    "Timestamp": "2020-05-02 03:08:35.270 +00:00",
    "Computer": "UCHIHA-PC.hinokabe.local",
    "Channel": "PwSh",
    "EventID": 4104,
    "Level": "info",
    "RecordID": 12519,
    "RuleTitle": "PwSh Scriptblock",
    "Details": {
        "ScriptBlock": "@{\\r\\nGUID = 'bd4390dc-a8ad-4bce-8d69-f53ccf8e4163'\\r\\nAuthor = 'Thomas Freudenberg'\\r\\nDescription = 'Powershell module for creating and extracting 7-Zip archives'\\r\\nCompanyName = 'N/A'\\r\\nCopyright = '2013-2020 Thomas Freudenberg'\\r\\nDotNetFrameworkVersion = '4.6.1'\\r\\nModuleVersion = '1.10.0.0'\\r\\nPowerShellVersion = '2.0'\\r\\nPrivateData = @{\\r\\n    PSData = @{\\r\\n        Tags = @('powershell', '7zip', '7-zip', 'zip', 'archive', 'extract', 'compress', 'PSEdition_Desktop', 'Windows')\\r\\n        LicenseUri = 'https://github.com/thoemmi/7Zip4Powershell/blob/master/LICENSE'\\r\\n        ProjectUri = 'https://github.com/thoemmi/7Zip4Powershell'\\r\\n        IconUri = 'https://raw.githubusercontent.com/thoemmi/7Zip4Powershell/master/Assets/7zip4powershell.png'\\r\\n        RequireLicenseAcceptance = $false\\r\\n        # ReleaseNotes = ''\\r\\n    } # End of PSData hashtable\\r\\n}\\r\\n\\r\\nNestedModules = @(\"7Zip4PowerShell.dll\")\\r\\nCmdletsToExport = @(\\r\\n    \"Expand-7Zip\",\\r\\n    \"Compress-7Zip\",\\r\\n    \"Get-7Zip\",\\r\\n    \"Get-7ZipInformation\")\\r\\n}\\r\\n"
    }
}

```

**Comment by YamatoSecurity** [comment]
@hitenkoku Thanks for looking into this for me.
That is weird. I am still getting only one space..
`./hayabusa-2.5.0-mac-intel json-timeline -f ./WELA-challenge-2023-05-13.json -o test.json -U -J`

`cat test.json |  grep Expand-7Zip | jq 'select (.RecordID == 12519)'`
```
{
  "Timestamp": "2020-05-02 03:08:35.270 +00:00",
  "Computer": "UCHIHA-PC.hinokabe.local",
  "Channel": "PwSh",
  "EventID": 4104,
  "Level": "info",
  "RecordID": 12519,
  "RuleTitle": "PwSh Scriptblock",
  "Details": {
    "ScriptBlock": "@{\\r\\nGUID = 'bd4390dc-a8ad-4bce-8d69-f53ccf8e4163'\\r\\nAuthor = 'Thomas Freudenberg'\\r\\nDescription = 'Powershell module for creating and extracting 7-Zip archives'\\r\\nCompanyName = 'N/A'\\r\\nCopyright = '2013-2020 Thomas Freudenberg'\\r\\nDotNetFrameworkVersion = '4.6.1'\\r\\nModuleVersion = '1.10.0.0'\\r\\nPowerShellVersion = '2.0'\\r\\nPrivateData = @{\\r\\n PSData = @{\\r\\n Tags = @('powershell', '7zip', '7-zip', 'zip', 'archive', 'extract', 'compress', 'PSEdition_Desktop', 'Windows')\\r\\n LicenseUri = 'https://github.com/thoemmi/7Zip4Powershell/blob/master/LICENSE'\\r\\n ProjectUri = 'https://github.com/thoemmi/7Zip4Powershell'\\r\\n IconUri = 'https://raw.githubusercontent.com/thoemmi/7Zip4Powershell/master/Assets/7zip4powershell.png'\\r\\n RequireLicenseAcceptance = $false\\r\\n # ReleaseNotes = ''\\r\\n } # End of PSData hashtable\\r\\n}\\r\\n\\r\\nNestedModules = @(\"7Zip4PowerShell.dll\")\\r\\nCmdletsToExport = @(\\r\\n \"Expand-7Zip\",\\r\\n \"Compress-7Zip\",\\r\\n \"Get-7Zip\",\\r\\n \"Get-7ZipInformation\")\\r\\n}\\r\\n"
  }
}
```

**Comment by YamatoSecurity** [comment]
I also tested on Windows but the same only 1 whitespace.
<img width="1782" alt="Screen Shot 2023-05-14 at 8 08 17 PM" src="https://github.com/Yamato-Security/hayabusa/assets/71482215/1d4ab403-e88f-49bf-a0bc-3fa5245964d3">

**Comment by YamatoSecurity** [comment]
@hitenkoku Yes, this is a low priority issue so don't worry about it for now.

**Comment by fukusuket** [comment]
@YamatoSecurity @hitenkoku 
I confirmed it reproduces in ver `2.5.0`, but it doesn't reproduce in ver 2.5.1.
I think it was probably fixed by #1043 ? What do you think?🤔

**Comment by YamatoSecurity** [comment]
@fukusuket Oh great! I just checked and it does seem to have been fixed in 2.5.1!
I will close this issue then.
